src/components/Bet.js

```diff
@@ -11,7 +11,7 @@ export default function Bet({
     <div className={`flex shadow-sm rounded-md ${className}`}>
       <div className="flex-1 flex items-center justify-between border border-gray-200 bg-white rounded-r-md truncate">
         <div className="flex-1 px-4 py-2 text-sm leading-5 truncate">
-          <a href={issue.url} target="_blank" className="text-gray-900 font-medium hover:text-gray-600 transition ease-in-out duration-150" title={issue.title}>{issue.title}</a>
+          <a href={issue.url} target="_blank" className="text-gray-900 font-medium hover:text-gray-600 transition ease-in-out duration-150" title={issue.title} rel="noreferrer">{issue.title}</a>
```

Comment:
@fmvilas curious to know why we don't want referrer links

Reply:
Eslint was complaining that this is a security issue not to have it. See https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md